localization/english/country_flavor_text_l_english.yml

```diff
@@ -3,8 +3,12 @@
  REV_GENERIC_FLAVOR_TEXT:1 "Dissatisfied with the direction that the so-called organized government was taking, these revolutionaries have decided to take up arms to overthrow it. Will they succeed in seizing power or will they be crushed like so many forgotten revolutions of the past?"
```

Comment:
Duplicate localizations
[12:25:19][pdx_localize.cpp:279]: Duplicate localization key. Key 'IRQ_FLAVOR_TEXT' is defined in both 'localization/english/country_flavor_text_l_english.yml' and 'localization/english/country_flavor_text_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'DRV_ADJ' is defined in both 'localization/english/dvg_countries_l_english.yml' and 'localization/english/dvg_countries_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'Kiya' is defined in both 'localization/english/dvg_flavor_persia_l_english.yml' and 'localization/english/dvg_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_BAJA_CALIFORNIA_city' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_BAJA_CALIFORNIA_port' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_BAJA_CALIFORNIA_mine' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_BAJA_CALIFORNIA_farm' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_BAJA_CALIFORNIA_wood' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_DURANGO_city' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_DURANGO_mine' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_DURANGO_farm' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_DURANGO_wood' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_RIO_GRANDE_city' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_RIO_GRANDE_port' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_RIO_GRANDE_mine' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_RIO_GRANDE_farm' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_RIO_GRANDE_wood' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_ZACATECAS_city' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_ZACATECAS_mine' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_ZACATECAS_farm' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_ZACATECAS_wood' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_BAJIO_city' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_BAJIO_mine' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_BAJIO_farm' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_BAJIO_wood' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_OAXACA_city' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_OAXACA_farm' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_OAXACA_mine' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_OAXACA_wood' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_OAXACA_port' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_SINALOA_city' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_SINALOA_farm' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_SINALOA_mine' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_SINALOA_wood' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_SINALOA_port' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_JALISCO_city' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_JALISCO_port' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_JALISCO_mine' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_JALISCO_farm' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_JALISCO_wood' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_GUERRERO_city' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_GUERRERO_port' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_GUERRERO_mine' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_GUERRERO_farm' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_GUERRERO_wood' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_MEXICO_city' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_MEXICO_mine' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_MEXICO_farm' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_MEXICO_wood' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_VERACRUZ_city' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_VERACRUZ_port' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_VERACRUZ_mine' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_VERACRUZ_farm' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_VERACRUZ_wood' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_BAHIA_city' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_BAHIA_port' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_BAHIA_mine' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_BAHIA_farm' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_BAHIA_wood' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_SANTA_CATARINA_city' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_SANTA_CATARINA_port' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_SANTA_CATARINA_mine' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_SANTA_CATARINA_farm' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_SANTA_CATARINA_wood' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'STATE_BAJA_CALIFORNIA' is defined in both 'localization/english/dvg_states_l_english.yml' and 'localization/english/map/states_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'STATE_RIO_GRANDE' is defined in both 'localization/english/dvg_states_l_english.yml' and 'localization/english/map/states_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'STATE_GUERRERO' is defined in both 'localization/english/dvg_states_l_english.yml' and 'localization/english/map/states_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'STATE_DURANGO' is defined in both 'localization/english/dvg_states_l_english.yml' and 'localization/english/map/states_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'STATE_JALISCO' is defined in both 'localization/english/dvg_states_l_english.yml' and 'localization/english/map/states_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'STATE_BAHIA' is defined in both 'localization/english/dvg_states_l_english.yml' and 'localization/english/map/states_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'STATE_SANTA_CATARINA' is defined in both 'localization/english/dvg_states_l_english.yml' and 'localization/english/map/states_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'STATE_BAJIO' is defined in both 'localization/english/dvg_states_l_english.yml' and 'localization/english/map/states_l_english.yml'